paper/paper.md

```diff
@@ -17,23 +17,23 @@ bibliography: paper.bib
 
-One suggestion to adequately calculate fluxes for compounds with low SNR is to first calculate the time lag for a *reference* compound with high SNR (e.g. CO~2~) and then apply the same time lag to the *target* compound of interest (e.g. N~2~O), with both compounds being recorded by the same analyzer [@Nemitz2018]. `DYCO` follows up on this suggestion by facilitating the dynamic lag-detection between the turbulent wind data and a *reference* compound and the subsequent application of found *reference* time lags to one or more *target* compounds. 
+One method to adequately calculate fluxes for compounds with low SNR is to first calculate the time lag for a *reference* compound with high SNR (e.g., CO~2~) and then to apply the same time lag to the *target* compound of interest (e.g., N~2~O), with both compounds being recorded by the same analyzer [@Nemitz2018]. `DYCO` uses this method by facilitating the dynamic lag-detection between the turbulent wind data and a *reference* compound and the subsequent application of found *reference* time lags to one or more *target* compounds. 
 
 
 
 # Statement of need
 
-The lag detection between the turbulent departures of measured wind and the scalar of interest is a central step in the calculation of EC ecosystem fluxes. In case the covariance maximization fails to detect a clear peak in the covariance function between the wind and the scalar, current flux calculation software can apply a constant default (nominal) time lag to the respective scalar. However, both the detection of clear covariance peaks in a pre-defined time window and the definition of a reliable default time lag is challenging for compounds which are often characterized by low SNR, such as N~2~O. In addition, the application of one static default time lag may produce inaccurate results in case systematic time shifts are present in the raw data.
+Lag detection between the turbulent departures of measured wind and the scalar of interest is a central step in the calculation of EC ecosystem fluxes. In the covariance maximization fails to detect a clear peak in the covariance function between the wind and the scalar, current flux calculation software can apply a constant default (nominal) time lag to the respective scalar. However, both the detection of clear covariance peaks in a pre-defined time window and the definition of a reliable default time lag is challenging for compounds that are often characterized by low SNR, such as N~2~O. In addition, the application of one static default time lag may produce inaccurate results when systematic time shifts are present in the raw data.
```

Comment:
"In case the covariance maximization fails" was changed to "In the covariance maximization fails" in the pull request, I think this is a typo. Please change to "If the covariance maximization fails".

Reply:
right - thanks

Reply:
now fixed